README.md

```diff
@@ -3,13 +3,25 @@
 
 This repository contains instructions to build modern Cloud-based Air Traffic Control using IBM Bluemix.
 
-Air Traffic Control receives flight information from Raspberry Pi powered ADS-B Ground Stations with Software Defined Radio(SDR) to receive ADS-B messages directly from commercial flights and publish MQTT messages to the IBM IoT Platform running in IBM Bluemix. It also supports a Swift-based iOS app to track flights using Augmented Reality toolkit by receiving MQTT messages from the IoT Platform.
+The Air Traffic Control application receives flight information from a Raspberry Pi powered ADS-B Ground Stations with Software Defined Radio(SDR) to receive ADS-B messages directly from commercial flights and publish MQTT messages to the IBM IoT Platform running in IBM Bluemix. It also supports a Swift-based iOS app to track flights using the Augmented Reality toolkit by receiving MQTT messages from the IoT Platform. The app will display all the flights traveling point to point within the range of the receiver.
 
-## Raspberry Pi powered ADS-B Ground Station
+With the advances in the field of avionics and the availability of cheap computing resources such as Raspberry Pi (RPi), one can very easily build a state­ of­ the­ art Ground Station. These Ground Stations can be replicated trivially using virtualization technologies such as Docker to be able to cover large swathes of areas. The RPi­powered Ground Stations, scattered all over the world, will do the following:
+* Use a SDR receiver with an antenna to receive information about flights that are in approximately 100­150 miles radius depending on the altitude and the line­of­sight.
```

Comment:
It should be `100-150` instead of `100150`. Also, `lineofsight` should be `line of sight`. Can you please comb through the entire README.md to make sure that there are no typos?